### ISSUE TITLE
PDF viewer shows useless "Open link" options

### ISSUE THREAD

**Issue by sopoforic** [burtonator/polar-bookshelf#1488]
2.0 beta 25 on Chrome 84.

The vertical ellipsis menu at the top right of the PDF viewer shows (disabled) options for "Open link in browser" and "Copy link to clipboard", which I presume apply to captured webpages. They should not be displayed when viewing a PDF.

**Comment by burtonator** [comment]
The issue is that a PDF *can* have it bug if it has a link I might decide to hide it instead of disabling it.

**Comment by sopoforic** [comment]
Ah, if you add the PDF with the save to polar extension, for example?

It might be worth exposing that property to the user, actually, even in case the thing was uploaded manually.

**Comment by burtonator** [comment]
You mean allow the user to set the property?  We're working on metadata post 2.1 but might add a simple setter dialog now...

**Comment by sopoforic** [comment]
Right. I have a big library of PDFs; I can imagine it being worthwhile to set a source link for some, in case I want to grab and share the link. It's not more urgent than other metadata, just something to consider.
